### PR TITLE
add Yoruba NER dataset

### PR DESCRIPTION
**Pull request by dadelani**
Added Yoruba GV dataset based on this paper

### CODE REVIEW

File: datasets/hausa_voa_ner/hausa_voa_ner.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+_TRAINING_FILE = "train_clean.tsv"
+_DEV_FILE = "dev.tsv"
+_TEST_FILE = "test.tsv"
+
+
+class HausaVoaNerConfig(datasets.BuilderConfig):
+    """BuilderConfig for Conll2003"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig forConll2003.
```
Comment (lhoestq) [inline datasets/hausa_voa_ner/hausa_voa_ner.py:74]:
```suggestion
    """BuilderConfig for HausaVoaNer"""

    def __init__(self, **kwargs):
        """BuilderConfig for HausaVoaNer.
```

File: datasets/yoruba_gv_ner/yoruba_gv_ner.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+_TRAINING_FILE = "train.tsv"
+_DEV_FILE = "valid.tsv"
+_TEST_FILE = "test.tsv"
+
+
+class YorubaGvNerConfig(datasets.BuilderConfig):
+    """BuilderConfig for Conll2003"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig forConll2003.
```
Comment (lhoestq) [inline datasets/yoruba_gv_ner/yoruba_gv_ner.py:73]:
```suggestion
class YorubaGvNerConfig(datasets.BuilderConfig):
    """BuilderConfig for YorubaGvNer"""

    def __init__(self, **kwargs):
        """BuilderConfig for YorubaGvNer.
```

**Review by lhoestq — APPROVED** [review]
Awesome thank you !

This PR includes two datasets though. It's ok this time, but in the future you could try to have one PR = one dataset please ?

**Comment by dadelani** [comment]
Thank you. Okay, other pull requests only have one dataset

**Comment by lhoestq** [comment]
The `RemoteDatasetTest` error in the CI is just a connection error, we can ignore it

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Comment by dadelani** [comment]
Thank you very much

**Pull request merged**